test/e2e/suite/serving/cainjector.go

```diff
@@ -155,7 +157,7 @@ var _ = framework.CertManagerDescribe("CA Injector", func() {
 				By("changing the name of the corresponding secret in the cert")
 				secretName := types.NamespacedName{Name: cert.Spec.SecretName, Namespace: f.Namespace.Name}
 				cert.Spec.DNSNames = append(cert.Spec.DNSNames, "something.com")
-				Expect(f.CRClient.Update(context.Background(), &cert)).To(Succeed())
+				Eventually(func() error { return f.CRClient.Update(context.Background(), &cert) }, "10s", "2s").Should(Succeed())
```

Comment:
To work around `Operation cannot be fulfilled on certificates.cert-manager.io "serving-certs": the object has been modified; please apply your changes to the latest version and try again
test/e2e/suite/serving/cainjector.go:160` errors in the tests: https://prow.build-infra.jetstack.net/view/gcs/jetstack-logs/pr-logs/pull/jetstack_cert-manager/3838/pull-cert-manager-e2e-v1-16/1379410202536710144